### PR TITLE
TO NOT BACKPORT - LPS-89689 obtain the fileName from the sourceFileName not the title

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**

### CODE REVIEW

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry.jsp [hunk 1]
```diff
@@ -313,7 +313,9 @@ renderResponse.setTitle(headerTitle);
 
 					<%@ include file="/document_library/edit_file_entry_picker.jspf" %>
 
-					<aui:input label="name" name="title" />
+					<aui:input label="title" name="title" />
+
+					<aui:input label="file-name" name="fileName" />
```
Comment (AliciaGarciaGarcia) [inline modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry.jsp:318]: It's needed that the field name be same that the DLFileVersion field

In this case it's not mandatory

**Comment by AliciaGarciaGarcia** [comment]
ci:test:stable

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
Just started reviewing :)

**Comment by AliciaGarciaGarcia** [comment]
see https://github.com/liferay-lima/liferay-portal/pull/987

**Pull request closed without merging**